www/src/pages/components/forms.js

```diff
@@ -330,6 +334,75 @@ export default withLayout(function FormControlsSection({ data }) {
       </p>
       <ReactPlayground codeText={RangeCustom} />
 
+      <LinkedHeading h="3" id="forms-custom-file">
+        File
+      </LinkedHeading>
+      <p>A custom styled File uploader.</p>
+      <Callout>
+        The custom <code>FormFile</code> will by default not visibly display
+        your selected file. This requires additional JS. The recommended plugin
+        to animate custom file input is{' '}
+        <a href="https://www.npmjs.com/package/bs-custom-file-input">
+          bs-custom-file-input
+        </a>
+        .
+      </Callout>
+      <ReactPlayground codeText={File} />
+
+      <h4>Translating or customizing the strings with HTML</h4>
+      <p>
+        Bootstrap also provides a way to translate the “Browse” text in HTML
+        with the <code>data-browse</code> attribute which can be added to the
+        custom input label (example in Dutch):
+      </p>
+      <Callout>
+        Note that the <code>data-browse</code> attribute does not to anything
+        unless the <code>custom</code> prop is set.
+      </Callout>
+      <ReactPlayground codeText={FileButtonTextHTML} />
+
+      <h4>Translating or customizing the strings with SCSS</h4>
+      <p>
+        Please refer to the official{' '}
+        <a href="https://getbootstrap.com/docs/4.4/components/forms/#translating-or-customizing-the-strings-with-scss">
+          Bootstrap documentation for translating via SCSS
+        </a>
+        . The <code>lang</code> prop can be used to pass the language.
+      </p>
+      <ReactPlayground codeText={FileButtonTextSCSS} />
```

Comment:
i think per how we do SCSS, this should be `FileButtonTextScss`, and the file should also be named that way